extension.ts

```diff
@@ -0,0 +1,65 @@
+import { Facet, Extension, RangeSet } from "@codemirror/state"
+import { WidgetType, EditorView, Decoration, ViewPlugin, DecorationSet, ViewUpdate, MatchDecorator } from "@codemirror/view"
+
+interface MorgenPluginSettings {
+	decorateIDs: 'show' | 'hide' | 'replace_with_emoji'
+}
+
+const DEFAULT_SETTINGS: MorgenPluginSettings = {
+	decorateIDs: 'show'
+}
+
+class IDWidget extends WidgetType {
+	constructor(readonly id: string) { super() }
+
+	toDOM() {
+		const wrap = document.createElement("span")
+		wrap.setAttribute("aria-label", `ID: ${this.id}`)
+		const box = wrap.appendChild(document.createElement("span"))
+		box.innerText = "▫️";
```

Comment:
Why a small box?

Reply:
Just chose it quite arbitrarily. I think we can update it as we get feedback. One guy already said he wanted 🟧  or 🟠 so I have no idea what people will prefer. They can also just hide it to 🤷